api/shareable.py

```diff
@@ -0,0 +1,55 @@
+from bson.objectid import ObjectId
+
+from pydantic import BaseModel
+from pymongo import MongoClient
+
+from loguru import logger
+
+
+class PlanData(BaseModel):
+    json_string: str
+    validation_levels: list[int]
+
+
+class MongoDBConnection:
+    def __init__(self):
+        self.connection = None
```

Comment:
Is this used?

Reply:
fixed the Dependency injection for MongoDB. Thanks for pointing this out.